heron/storm/src/java/org/apache/storm/Config.java

```diff
@@ -260,6 +260,53 @@
   public static final String TOPOLOGY_PROJECT_NAME = "topology.project.name";
 
   /**
+   * A list of hosts of ZooKeeper servers used to manage the cluster.
+   */
+  public static final String STORM_ZOOKEEPER_SERVERS = "storm.zookeeper.servers";
+
+  /**
+   * The port Storm will use to connect to each of the ZooKeeper servers.
+   */
+  public static final String STORM_ZOOKEEPER_PORT = "storm.zookeeper.port";
+
+  /**
+   * The root directory in ZooKeeper for metadata about TransactionalSpouts.
+   */
+  public static final String TRANSACTIONAL_ZOOKEEPER_ROOT="transactional.zookeeper.root";
```

Comment:
umm, to keep uniformity, should we surround assignment operator using whitespace?